themes/bootstrap3/templates/collection/view.phtml

```diff
@@ -25,42 +25,58 @@
 ?>
 
 <?php if (isset($this->scrollData) && ($this->scrollData['previousRecord'] || $this->scrollData['nextRecord'])): ?>
-  <ul class="pager">
-    <?php if ($this->scrollData['previousRecord']): ?>
-      <?php if ($this->scrollData['firstRecord']): ?>
+  <nav aria-label="<?=$this->transEsc('pagination_navigation')?>">
+    <ul class="pager hidden-print">
+      <?php if ($this->scrollData['previousRecord']): ?>
+        <?php if ($this->scrollData['firstRecord']): ?>
+          <li>
+            <a href="<?=$this->recordLink()->getUrl($this->scrollData['firstRecord'])?>" title="<?=$this->transEsc('First Search Result')?>" rel="nofollow"  aria-label="<?=$this->transEsc('go_to')?> <?=$this->transEsc('First Search Result')?>">
```

Comment:
Is all this scrollData logic exactly duplicated from the record/view.phtml? If so, I wonder if it would make sense to refactor it to a sub-template so we don't have to remember to keep two versions in sync.